BlazorApp1/wwwroot/demo1.json

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
please rename this json file based on the chart it is connected to

Reply:
done.